_header.scss

```diff
@@ -0,0 +1,194 @@
+@font-face {
+  font-family: "Andalan Eug";
+  src: url("fonts/Andalan-Eug.woff2") format("woff2"),
+    url("fonts/Andalan-Eug.woff") format("woff");
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+$background: #eae7dc;
+$dark-transparent: rgb(29, 26, 26, 0.75);
+$headline-font: "Andalan Eug";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Open Sans";
+  font-size: 18px;
+}
+body {
+  margin: 0;
+  padding: 0;
+  background-color: $background;
+}
```

Comment:
We already have the same lines in the _basic.scss file (in the * and body part ), also the fonts are already added, so lines 1-24 should be reviewed to see what is really necessary .